example-app/src/main/resources/application.properties

```diff
@@ -2,8 +2,13 @@
 #
 eip.home=${user.home}${file.separator}.openmrs-eip
 
-#eip.watchedTables=person
-#camel.springboot.main-run-controller=true
+camel.springboot.routes-include-pattern=classpath:camel/*,classpath:camel-template/*,classpath:camel-rest/*
```

Comment:
I guess this will be xml files provided? In which case we'd have to worry about that end with the `*.xml` extension?